demos/formula_recognition_demo/python/utils.py

```diff
@@ -86,7 +86,7 @@ def create_renderer():
     of latex formula could be performed
     """
     command = subprocess.run(["pdflatex", "--version"], stdout=subprocess.DEVNULL,
-                             stderr=subprocess.DEVNULL, check=False)
+                             stderr=subprocess.DEVNULL, check=False, shell=True)
```

Comment:
What is this for? My assumption was it is to address @vladimir-dudnik's issue of running on Windows, but I've rechecked and the demo works with `shell=False`.

Reply:
@Wovchena I've got a problem running demo on Windows without Latex package installed.

Reply:
Ok, please explain that in code comment

Reply:
@Wovchena done